profiles/killbill/src/main/java/org/killbill/billing/server/listeners/CleanupListener.java

```diff
@@ -76,5 +77,9 @@ public void contextDestroyed(final ServletContextEvent servletContextEvent) {
                 logger.warn("Unable to de-register driver", e);
             }
         }
+
+//         avoid memory leaks: https://logback.qos.ch/manual/jmxConfig.html
+        LoggerContext lc = (LoggerContext) LoggerFactory.getILoggerFactory();
+        lc.stop();
```

Comment:
@laurentbh Did you verify that, if `jmxConfigurator` isn't set in the `logback.xml`, the `stop` here is a no-op?

Reply:
yes, no exception raised when jmx is not enable in`logback.xml`